Emby.Server.Implementations/ScheduledTasks/TaskManager.cs

```diff
@@ -19,8 +20,6 @@ public class TaskManager : ITaskManager
     {
         public event EventHandler<GenericEventArgs<IScheduledTaskWorker>> TaskExecuting;
```

Comment:
What about this one?

Reply:
I only migrated some of the events in this PR, generally the ones that had already been at least partially moved to the custom event system. I'm working on followup PR(s) to migrate the rest of the events, to keep the size of each individual PR within reason.